src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts

```diff
@@ -805,4 +794,37 @@ describe('DotEditContentComponent', () => {
             });
         });
     });
+
+    describe('Auto save', () => {
+        it('should call the save endpoint after a model change happens', () => {
+            const model: DotPageContainer[] = [
+                {
+                    identifier: '1',
+                    uuid: '2',
+                    contentletsId: ['3', '4']
+                }
+            ];
+
+            const newModel: DotPageContainer[] = [
+                {
+                    identifier: '2',
+                    uuid: '3',
+                    contentletsId: ['4', '5']
+                }
+            ];
+
+            let dotEditPageService: DotEditPageService;
+            dotEditPageService = de.injector.get(DotEditPageService);
+
+            spyOn(dotEditPageService, 'save').and.returnValue(Observable.of(true));
+            spyOn(dotEditContentHtmlService, 'getContentModel').and.returnValue({});
+            fixture.detectChanges();
+
+            dotEditContentHtmlService.pageModelChange.next(model);
+            fixture.detectChanges();
+
+            dotEditContentHtmlService.pageModelChange.next(newModel);
+            expect(dotEditPageService.save).toHaveBeenCalled();
```

Comment:
I think this  ```dotEditPageService.save``` it's being called twice because there is 2 ```.pageModelChange.next(model);``` so we can do ```toHaveBeenCalledTimes(2)```